dockerfiles/centos/is/README.md

```diff
@@ -21,7 +21,7 @@ git clone https://github.com/wso2/docker-is.git
 
 - Navigate to `<IS_DOCKERFILE_HOME>` directory. <br>
   Execute `docker build` command as shown below.
-    + `docker build -t wso2is:5.11.0-centos .`
+    + `docker build -t wso2is:6.0.0-centos .`
```

Comment:
Do we need centos docker files

Reply:
Checking with the team. Will adhere to that.